packages/dds/tree/src/test/shared-tree-core/sharedTreeCore.spec.ts

```diff
@@ -65,7 +79,7 @@ describe("SharedTreeCore", () => {
             assert.equal(Object.entries(indexSummaryTree.tree).length, indexes.length,
                 "Expected both indexes to be present in the index subtree of the summary");
 
-            assert.equal(stats.treeNodeCount, 2,
+            assert.equal(stats.treeNodeCount, 4,
```

Comment:
Why did this change exaclty?

Reply:
I changed the summary of MockIndex to use the createSingleBlobSummary helper. Most DDS_s use that for their summaries, so I thought it would be good to have the MockIndex follow the same pattern to demonstrate that you can "summarize an index just like you would summarize a DDS". But I guess the format used by createSingleBlobSummary is a little less compact than the custom tree I was making before; I'll look at the implementation of createSingleBlobSummary and double check that it's expected.

Reply:
Sounds good. Feel free to resolve this afterwards.

Reply:
Actually, the count before was just wrong (I didn't bump it when making the tree). Another reason why using the battle-tested helper is good! :)